### PR TITLE
Php7.1, updated coding style, removed deprecated AbstractCommand

### LINKED ISSUE #13: Bump PHP 7.1

### PR DESCRIPTION
**Pull request by mabar**
Closes #13

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,18 +18,23 @@
     }
   ],
   "require": {
-    "php": ">= 5.6",
+    "php": ">= 7.1",
     "symfony/console": "^3.2.0 || ^4.0.0"
```
Comment (f3l1x) [inline composer.json]: I would drop 3.x console.

File: ruleset.xml [hunk 2]
```diff
@@ -1,9 +1,11 @@
 <?xml version="1.0"?>
 <ruleset name="Contributte">
-    <!-- Extending rulesets -->
-    <rule ref="vendor/ninjify/coding-standard/ruleset.xml"/>
+    <!-- Contributte Coding Standard -->
+	<rule ref="./vendor/ninjify/coding-standard/contributte.xml">
+		<exclude name="SlevomatCodingStandard.Operators.DisallowIncrementAndDecrementOperators.DisallowedPostIncrementOperator"/>
```
Comment (f3l1x) [inline ruleset.xml]: Is this rule used here?

File: src/Helper/ContainerHelper.php [hunk 3]
```diff
@@ -1,13 +1,11 @@
...
 
 use Nette\DI\Container;
 use Symfony\Component\Console\Helper\Helper;
 
 /**
- * @author Milan Felix Sulc <sulcmil@gmail.com>
- * @author Filip Procházka <filip@prochazka.su>
  * @deprecated
  */
 class ContainerHelper extends Helper
```
Comment (f3l1x) [inline src/Helper/ContainerHelper.php]: Yep, drop it.

File: ruleset.xml (hunk 2 above)
Comment (mabar) [reply ruleset.xml]: Yes, here https://github.com/contributte/console/blob/master/src/DI/ConsoleExtension.php#L107

Comment (mabar) [reply ruleset.xml]: I could rewrite foreach to use array keys instead

Comment (f3l1x) [reply ruleset.xml]: It's OK. Thanks.

File: src/DI/ConsoleExtension.php [hunk 4]
```diff
@@ -57,39 +48,38 @@ public function __construct($cliMode = FALSE)
...
 	 */
-	public function loadConfiguration()
+	public function loadConfiguration(): void
 	{
 		$builder = $this->getContainerBuilder();
 		$config = $this->validateConfig($this->defaults);
 
 		// Skip if isn't CLI
-		if ($this->cliMode !== TRUE) return;
+		if ($this->cliMode !== true)
```
Comment (f3l1x) [inline src/DI/ConsoleExtension.php]: Could you please keep it on same row?

File: src/DI/ConsoleExtension.php [hunk 5]
```diff
@@ -121,21 +111,20 @@ public function loadConfiguration()
...
 	 */
-	public function beforeCompile()
+	public function beforeCompile(): void
 	{
 		$builder = $this->getContainerBuilder();
 		$config = $this->validateConfig($this->defaults);
 
 		// Skip if isn't CLI
-		if ($this->cliMode !== TRUE) return;
+		if ($this->cliMode !== true)
```
Comment (f3l1x) [inline src/DI/ConsoleExtension.php]: Same above

Comment (mabar) [reply src/DI/ConsoleExtension.php]: I always accidentally run autoformating and not found yet how to disable that in phpstorm, sorry

Comment (f3l1x) [reply src/DI/ConsoleExtension.php]: No problem ;-)

**Pull request merged**